packages/react-server/src/webpack-plugin/webpack-plugin.ts

```diff
@@ -92,6 +94,7 @@ function serverSource(options: Options, compiler: Compiler) {
       port: ${port},
       ip: ${JSON.stringify(host)},
       assetPrefix: ${JSON.stringify(assetPrefix)},
+      proxy: ${JSON.stringify(proxy)},
```

Comment:
dont think we need to JSON.stringify this option, otherwise we will always get string instead boolean value.